lib/verify.go

```diff
@@ -109,6 +111,30 @@ func caBundle(caPath string) *x509.CertPool {
 	return bundle
 }
 
+func fetchAIA(cert *x509.Certificate) *x509.Certificate {
+	for _, url := range cert.IssuingCertificateURL {
```

Comment:
rfc4325 says we should look at the file extension to see if it's .cer or .p7c.  We could return if it's .p7c and only support .cer fetching